README.md

```diff
@@ -115,6 +115,12 @@ make dev-compose-up-linux
 docker run -e SURVEY_RUNNER_SCHEMA_URL=http://docker.for.mac.host.internal:5000 -it -p 8000:8000 onsdigital/eq-questionnaire-launcher:latest
```

Comment:
We need to set SDS API URL the same way here?.

Reply:
Added 👍